Adicionar novos comandos no Makefile

bruno-s-campos opened the pull request:
## Descrição

Adicionar os comandos abaixo no Makefile:

- **db-migrate**: executar uma migração.
```
make db-migrate
```
- **db-migrate-make**: criar uma nova migração.
```
make db-migrate ARGS=<nome da migração>
```
- **db-refresh**: executar todas os rollbacks e, em seguida, executar todas migrações.
```
make db-refresh
```
- **db-rollback**: executar um rollback.
```
make db-rollback
```
- **db-shell**: acessar o banco de dados pelo terminal.
```
make db-shell USER=<nome do usuário no banco de dados>
```
- **shell**: entrar no container do backend, permitindo assim acessar comandos do Artisan.
```
make shell 
```

## Cenários de teste

- Levantar o container do projeto (`make up-dev`);
- Executar cada um dos comandos listados acima e confirmar que nenhum erro ocorreu na execução deles.

Commit:
Adicionar novos comandos no Makefile

Novos comandos adicionados:
- db-migrate: executar uma migração.
- db-migrate-make: criar uma nova migração.
- db-refresh: executar todas os rollbacks e, em seguida, executar todas
migraçẽs.
- db-rollback: executar um rollback.
- db-shell: acessar o banco de dados pelo terminal.
- shell: entrar no container do backend, permitindo assim acessar
comandos do Artisan.

dirsouza reviewed (changes requested):
Acrescentar no comando shell a tag `--user=${USERNAMEDOCKER}` antes do nome do container `back,` afim de evitar problemas de permissões.
O mesmo vale para os demais comandos relacionados a manipulação do container `back`.

FernandoCostaB commented:
Esse db-refresh vai ser executado ? Pq  é perigoso ficar rodando ele sempre, principalmente em prod. Refresh ele apaga todas as tabelas e os dados...

Commit:
Realizar mudanças solicitadas na review

- Executar o comando `make shell` usando o usuário definido na variável
de ambiente USERNAMEDOCKER;
- Remover o comando db-refresh, para evitar riscos de alguém executá-lo
em produção e remover todas as tabelas.

Commit:
Atulizar a documentação do projeto

Adicionar os novos comandos no README do projeto.

dirsouza reviewed (approved):
Ok
